docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -223,77 +228,123 @@ draws from Bayesian-derived joint posterior distributions.
+implement and understand but can be less efficient and less flexible in representing variable state durations. As such, our team has had an ongoing discussion regarding updating Vivarium methods from 
+discrete time to discrete event simulation as we continue to seek more accurate and efficient ways to model complex systems.
 
 Calibration
 +++++++++++
 
-.. todo:: 
+Vivarium microsimulations are retrospectively assessed for quality through a process we refer to as verification and validation (or V&V). This process is crucial in ensuring our models match the 
+target values they were developed to replicate, and that our models are sensible and reflect trends in real-world data not used in model development. Target values come from input data (often from GBD
+estimates, but also other data sources) and typically include population age and sex/gender structure, risk factor exposures and effects, all-cause mortality rates, cause-specific parameters, 
+intervention coverages, and more. If any of these checks fail, we debug and go through the process again until model results behave as expected. To read more about our team's V&V process, see 
+`<vivarium_best_practices_results_processing>`_.
 
-    Describe how some microsimulations automatically fit to data as opposed to being retroactively checked against data like we do on SimSci with a separate V&V process. Read more about our V&V process
-    here [[insert link]]
+In contrast, the Framework for Reconstructing Epidemiological Dynamics (FRED) (an agent-based model) has a calibration process that is directly integrated into the simulation's runtime, 
+allowing the model to adjust its parameters dynamically to ensure a better fit with empirical data. Unlike the separate V&V process, where models are developed and then retrospectively compared and 
+adjusted based on data, FRED's integrated calibration allows for a more dynamic and responsive approach to fitting model outputs to real-world evidence. You can read more about FRED 
+`here <https://bmcpublichealth.biomedcentral.com/articles/10.1186/1471-2458-13-940>`__.
+
+.. note:: 
+
+  Although the authors of FRED do not explicitly name it as a microsimulation, they refer to it as an agent-based modeling system that represents every individual in a sepcific geographic region. 
+  Their use of a built-in calibration mechanism provides a useful contrast to the retrospective and iterative process utilized by Vivarium researchers.
 
 Correlation
 +++++++++++
 
-.. todo:: 
-
-    Describe how we overlay population-level statistics (e.g. GBD estimates) onto individuals rather than using individual-level data (e.g. hospital records) - can use CVD as an example of the latter.
+When developing a Vivarium simulation, we typically overlay population-level statistic generated through the GBD onto individual simulants, rather than using individual-level input data such 
```

Comment:
```suggestion
When developing a Vivarium simulation, we typically overlay population-level statistics generated through the GBD onto individual simulants, rather than using individual-level input data such 
```